src/Components/SavedSearchAlert/Components/AddFiltersScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Clickable, Flex, Join, Separator, Spacer, Text } from "@artsy/palette"
+import { useTransitionPanel } from "Components/TransitionPanel"
+import ChevronLeftIcon from "@artsy/icons/ChevronLeftIcon"
+import { AddFiltersRarity } from "Components/SavedSearchAlert/Components/AddFiltersScreenRarity"
+
+export const AddFiltersScreen: React.FC = () => {
+  const { navigateTo } = useTransitionPanel()
+  return (
+    <Flex flexDirection="column" width={345} height={464}>
```

Comment:
What is this height supposed to be?

Reply:
I believe we can remove the height

Reply:
And probably width as well
it will be "auto" by default

Reply:
It was just temporary, working on a different design now.